src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -33158,6 +33158,466 @@ class MyCollection : IEnumerable
                 """);
         }
 
+        [WorkItem("https://github.com/dotnet/roslyn/issues/72461")]
+        [Fact]
+        public void Add_ParamsArray_01()
+        {
+            string source = """
+                using System;
+                using System.Collections.Generic;
+                static class Extensions
+                {
+                    public static void Add<T>(this ICollection<T> collection, params T[] elements)
+                    {
+                        foreach (T element in elements)
+                            collection.Add(element);
```

Comment:
>Add [](http://example.com/codeflow?start=39&length=3)

Just curious, what method is called here?

Reply:
The implementation of `ICollection<T>.Add(T)` is called on the `collection` instance.

Reply:
> The implementation of `ICollection<T>.Add(T)` is called on the `collection` instance.

I that method is available and it is preferred over `params` version here, how do we end up using `params` method for collection expression?

Reply:
The `ICollection<KeyValuePair<K, V>>.Add()` method on `Dictionary<K, V>` is ignored in the collection expression because it is an explicit implementation.